src/octree/mod.rs

```diff
@@ -164,6 +164,37 @@ impl<'a> InternalIterator for PointsInBoxIterator<'a> {
     }
 }
 
+pub struct PointsInOrientedBeamIterator<'a> {
+    octree: &'a Octree,
+    beam: &'a OrientedBeam,
+    intersecting_nodes: Vec<NodeId>,
+}
+
+impl<'a> InternalIterator for PointsInOrientedBeamIterator<'a> {
+    fn size_hint(&self) -> Option<usize> {
```

Comment:
would it maybe be more useful to implement it already with the std Iterator trait?

Reply:
Yes, I'll have to adapt to Marco's change.

Reply:
Done.